wpeverest/user-registration
# Enhance - Seperate Captcha Settings for Login and Registration Form

### All Submissions:



### Changes proposed in this Pull Request:

Previously, Users were allowed to select only one captcha from settings for all registration forms as well as login and reset password form. Now, due to this enhancement users will be able to select different configured captchas for different registration as well as login and reset password form.

### How to test the changes in this Pull Request:

1. Goto User Registration > Settings > Captcha and validate backward compatibility. Similar for Form Settings.
2. Select different configured captchas for different registration forms in Form Settings > Enable Captcha and verify whether it is working or not.
3. Goto User Registration > Settings > General > Login Options and select Captcha type for Enable Captcha option and verify for login option.

### Types of changes:



### Other information:



### Changelog entry

> Enhance - Seperate Captcha Settings for Login and Registration Form
